README.md

```diff
@@ -34,33 +34,68 @@ Authrep is a 'one-shot' operation to authorize an application and report the ass
 
 Here is an example assuming that you are using the `app_id/app_key` authentication mode:
 ```javascript
+
 var Client = require('3scale').Client;
 
-// Creates a Client with default host and port.This will comunicate with the 3scale platform SaaS default server.
-client = new Client("your provider key");
+//Create a Client with a given host and port when connecting to an on-premise instance of the 3scale platform:
+client = new Client({host: "backend.example.com", port: 80});
```

Comment:
Guess what.... I'd use "service management API" as name instead of backend here also (and in the code :-) )